app/uk/gov/hmrc/fhregistrationfrontend/views/helpers/AddressInternational.scala.html

```diff
@@ -14,67 +14,95 @@
+@import uk.gov.hmrc.govukfrontend.views.html.components._
 
-@(formDate: Form[_], params: uk.gov.hmrc.fhregistrationfrontend.views.helpers.AddressUKParams)(implicit message: Messages)
+@import uk.gov.hmrc.fhregistrationfrontend.forms.definitions.ContactPersonForm.otherInternationalContactAddressKey
+@(formDate: Form[_], params: uk.gov.hmrc.fhregistrationfrontend.views.helpers.AddressUKParams, viewHelpers: ViewHelpers)(implicit message: Messages)
 
 @fieldName = @{
-  params.context match {
-    case Some(context) => context + "_" + params.name
-    case _ => params.name
-  }
+    params.context match {
+        case Some(context) => context + "_" + params.name
+        case _ => params.name
+    }
 }
-<fieldset id="@{fieldName}-fieldset">
-<legend class="form-label-bold">@{params.label}</legend>
+<fieldset id="@{fieldName}-fieldset" class="govuk-fieldset">
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">@{params.label}</legend>
+    @viewHelpers.govukInput(Input(
+        id = s"$fieldName.Line1",
+        name = s"$fieldName.Line1",
+        label = Label(
+            isPageHeading = false,
+            classes = "govuk-body",
+            content = HtmlContent(s"<strong>${message("fh.address.line1.label")}</strong>")
+        ),
+        spellcheck = Some(false),
+        autocomplete = Some("address-line1"),
+        classes = "govuk-!-width-three-quarters",
+        value = Some(s"${formDate(s"$fieldName.Line1").value.getOrElse("")}"),
+        errorMessage =
+            formDate(s"$fieldName.Line1").error.map{ e =>
+                ErrorMessage(
+                    content = Text(message(s"fh.contact_person.$fieldName.Line1.${e.message}")),
+                    visuallyHiddenText = Some(message("fh.generic.errorPrefix"))
+                )
+            }
+    ))
 
-@if(params.hasErrors) {
-  <span class="error-message" id="@{params.name}-inline-error">
-    @params.error
-  </span>
-}
-
-@TextInput(
-  TextInputParams(
-    name = "Line1",
-    context = Some(fieldName),
-    value = Some(s"${formDate(s"$fieldName.Line1").value.getOrElse("")}"),
-    label = Messages("fh.address.line1.label"),
-    error = Messages("fh.address.line1." + formDate(s"$fieldName.Line1").error.map(_.message).getOrElse("")),
-    hasErrors = formDate(s"$fieldName.Line1").hasErrors
-  )
-)
-
-@TextInput(
-  TextInputParams(
-    name = "Line2",
-    context = Some(fieldName),
-    value = Some(s"${formDate(s"$fieldName.Line2").value.getOrElse("")}"),
-    label = Messages("fh.address.line2_optional.label"),
-    error = Messages("fh.address.other.error"),
-    hasErrors = formDate(s"$fieldName.Line2").hasErrors
-  )
-)
+    @viewHelpers.govukInput(Input(
+        id = s"$fieldName.Line2",
+        name = s"$fieldName.Line2",
+        label = Label(
+            isPageHeading = false,
+            classes = "govuk-body",
+            content = HtmlContent(s"<strong>${message("fh.address.line2_optional.label")}</strong>")
```

Comment:
Same as above - use class instead of `strong`

Reply:
done here https://github.com/hmrc/fh-registration-frontend/pull/346/commits/ae071fbf2d4d7d90c10c52192b8a9191d03a40cc